pandas/core/nanops.py

```diff
@@ -353,10 +353,16 @@ def _nanvar(values, axis=None, skipna=True, ddof=1):
         values = values.copy()
         np.putmask(values, mask, 0)
 
-    X = _ensure_numeric(values.sum(axis))
-    XX = _ensure_numeric((values ** 2).sum(axis))
-    result = np.fabs((XX - X * X / count) / d)
-    return result
+    # This implementation is based on that of nanvar in numpy 1.8 and up. TODO:
+    # call directly into numpy when we bump numpy up 1.8 .
```

Comment:
I added the conditional to `nanvar` itself, but I just realized that I didn't check whether anything calls `_nanvar` directly. In any case, I don't have any strong preferences for putting the conditional in any particular place.